pycbc/events/events.py

```diff
@@ -308,22 +308,42 @@ def keep_near_injection(self, window, injections):
         i = indices_within_times(gpstime, inj_time - window, inj_time + window)
         self.events = self.events[i]
 
-    def keep_loudest_in_interval(self, window, num_keep):
+    def keep_loudest_in_interval(self, window, num_keep, log_chirp_width=None):
         if len(self.events) == 0:
             return
 
         e = self.events
         stat = newsnr(abs(e['snr']), e['chisq'] / e['chisq_dof'])
         time = e['time_index']
 
+        # convert time to time bin index
         wtime = (time / window).astype(numpy.int32)
         bins = numpy.unique(wtime)
 
+        if log_chirp_width:
+            from pycbc.conversions import mchirp_from_mass1_mass2
+            m1 = numpy.array([p['tmplt'].mass1 for p in self.template_params])
+            m2 = numpy.array([p['tmplt'].mass2 for p in self.template_params])
+
+            # chirp mass of each template
```

Comment:
this comment line & blank newline could be removed, calculation of chirp mass is self-documenting.